usagov/usagov-2021
# Usagov 998 fix amp link missing href

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-998

## Description


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



1. Clear the cache
2. Go to localhost/agency-index
3. Inspect the current letter in the menu and check that it looks like this 
Before:
<img width="960" alt="image" src="https://github.com/usagov/usagov-2021/assets/39020809/bdefae2e-583d-4109-97a0-8de6a2e05601">

After:
![image](https://github.com/usagov/usagov-2021/assets/39020809/782be753-50e9-44b0-8cac-2785aa8537d0)

4. Check that the menu works as expected and that the hover style is correct. 

### Requires New Config

### Requires New Content

### Validation Steps

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
4. Find the commit of this pull request.
5. Build and deploy the changes.
6. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
